cmd/tailscale/cli/serve.go

```diff
@@ -548,7 +552,13 @@ func printWebStatusTree(sc *ipn.ServeConfig, hp ipn.HostPort) {
 		case h.Proxy != "":
 			return "proxy", h.Proxy
 		case h.Text != "":
```

Comment:
We could make the Text field be `*string` instead

Reply:
I'm open to that. I was trying to avoid changing the entire backend to support a single empty case. This approach seems harmless if imprecise.

Reply:
I'm thinking about cases where users have automation that sets this up using the localapi.

The changes shouldn't be too much as json.Unmarshal should handle this correctly I think if we don't add omitempty.

Reply:
OK I'll rework it.

Reply:
PTAL